internal/metrix/meter.go

```diff
@@ -0,0 +1,190 @@
+// Package metrix implements stats-related functionality.
+package metrix
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
Would using OpenTelemetry be more flexible and will allow us to add more things in the future?

Reply:
It was discussed and decided against.